lib/remote/apilistener.cpp

```diff
@@ -430,11 +430,31 @@ void ApiListener::ListenerCoroutineProc(boost::asio::yield_context yc, const Sha
 
 	auto& io (IoEngine::Get().GetIoContext());
 
+	auto lastModified = static_cast<time_t>(-1);
```

Comment:
`s/auto/time_t/` and drop the cast. (The C++ people say "**almost** always auto" for a reason.)